packages/app/junit.xml

```diff
@@ -0,0 +1,449 @@
+<testsuites name="jest tests" tests="194" failures="0" time="15.198000000000002">
```

Comment:
What's this? 😀

Reply:
I was going to ask about this after I got the CI test results. I had some failing tests on circle-ci. I ran the recommended script, which led to this, but I don't see why I need this.

Reply:
I can remove it so you can see the error